src/components/AddBooks/AddBook.js

```diff
@@ -3,29 +3,37 @@ import axios from "axios";
 import Navigation from "../Navigation/Navigation";
 import Grid from "../Grid/Grid";
 
-const AddBooks = () => {
+const AddBooks = (props) => {
+  const {
+    setName,
+    verified,
+    setVerified,
+    displayCart,
+    setDisplayCart,
+    signOn,
+    setSignOn,
+    employee,
+    setEmployee,
+    users,
+    setUsers,
+    search: propSearch,
+    setSearch: propSetSearch,
+    books: propBooks,
+    setBooks: propSetBooks,
```

Comment:
we don't need to change these they are global states that work in multiple areas having them overwritten here doesn't allow the site to function properly